site/docs/layouts/partials/menu.html

```diff
@@ -18,7 +18,11 @@
                 {{- if .LinkTitle }}
                     {{- .LinkTitle }}
                 {{- else }}
-                    {{- replace .File.Dir .Parent.File.Dir "" | replaceRE "/" "-" }}{{- .File.TranslationBaseName }}
+                    {{- $noparent := .File.Dir }}
+                    {{- if .Parent.File }}
+                        {{- $noparent := replace .File.Dir .Parent.File.Dir "" }}
```

Comment:
```suggestion
                        {{- $noparent = replace .File.Dir .Parent.File.Dir "" }}
```

Reply:
I am not sure how this works in Hugo, but without this the top level files will have a preceding "-".

Reply:
Thanks. It looks like `:=` declares a new variable in the current scope (shadowing any enclosing scope) and `=` is an assignment to an existing variable. So the `if` block in my version sets and immediately discards a `$noparent` variable. Oops!

The Hugo documentation is... hard to use as a reference manual. But I finally tracked down what's going on in [Go template documentation](https://golang.org/pkg/text/template/#hdr-Variables).

Reply:
That was my initial thought, but as you said, following the documentation to find the exact specification is not easy.
Now that you mention it I see the paths of the sub folders having too much content, so I was only partial correct in my review change request :)